extensions/amp-accordion/amp-accordion.md

```diff
@@ -95,14 +95,20 @@ to preserve the collapsed/expanded state.[/filter]
 
 [/example]
 
+### Migrating from 0.1
+
+Unlike `0.1`, the experimental `1.0` version of `amp-accordion` does not currently support session states. It behaves as if the `disable-session-states` attribute is always set.
+
+Both `0.1` and `1.0` versions of `amp-accordion` can be used with `amp-bind`. However, in `1.0` `amp-bind` can bind directly to the `expanded` attribute whereas in `0.1` `amp-bind` operates on the `[data-expand]` attribute. The `[data-expand]` attribute is no longer used in `1.0`. See the `expanded` attribute in the next section for more information.
```

Comment:
```suggestion
The experimental `1.0` version of `amp-accordion` does not support session states. It behaves as if the `disable-session-states` attribute is always applied.

Version `0.1` and `1.0` are compatible with `amp-bind`, but some binding syntax is different. You may bind directly with the `expanded` attribute in version `1.0`. The `[data-expanded]` is not supported in version `1.0`. See the `expanded` attribute below for further information.
```

Reply:
Thanks, I used this suggestion in the latest revision.